l10n_it_account_tax_kind/__manifest__.py

```diff
@@ -7,10 +7,11 @@
 {
     'name': 'Italian Localisation - Natura delle aliquote IVA',
     'version': '12.0.2.1.0',
-    'development_status': 'Production/Stable',
+    'development_status': 'Beta',
```

Comment:
Tenderei a non fare il downgrade del `development_status` ma piuttosto promuovere i moduli `l10n_it_account` e `account_tax_balance` a `Production/Stable` (in riferimento all'errore che hai riportato su https://github.com/OCA/l10n-italy/pull/2893/commits/88f3e40962a38f8b2317975933326024b248e3a2), analogamente a quanto fatto su `14.0`:

- https://github.com/OCA/l10n-italy/pull/2473
- https://github.com/OCA/account-financial-reporting/pull/825

Reply:
Grazie della review!
Ho fatto la modifica qui e in:
- https://github.com/OCA/account-financial-tools/pull/1463
- https://github.com/OCA/account-financial-reporting/pull/903